ui/src/createAzureClient.ts

```diff
@@ -39,31 +39,10 @@ export function createGenericClient(clientInfo?: ServiceClientCredentials | { cr
         credentials = clientInfo;
     }
 
-    return new GenericServiceClient(credentials, <types.IMinimumServiceClientOptions>{
+    const gsc: typeof import('./GenericServiceClient') = await import('./GenericServiceClient');
```

Comment:
Did you test this import stuff with webpack? And no issues?

Reply:
Yes, and yes :smile: